freecad/gridfinity_workbench/grid_initial_layout.py

```diff
@@ -119,7 +119,7 @@ def __init__(self, obj: fc.DocumentObject, *, baseplate_default: bool) -> None:
 
         obj.setEditorMode("Baseplate", 2)
 
-    def make(self, obj: fc.DocumentObject) -> None:
+    def make(self, obj: fc.DocumentObject) -> list[list[bool]]:
```

Comment:
RectangleLayout is a data class, calling make on this class does not make sense at all. Returning a list of lists does not make sense. It is already happening, so we might need to create an issue for this and fix it in another PR, but this is just weird. It is not a feature.